### PR TITLE
Ready: Add com.kitware.F3D

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://gitlab.kitware.com/f3d/f3d/-/issues/226
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by barthalion** [comment]
The build failure means that either there's no desktop file or if it is, it has wrong `Icon=` set, or if it matches app ID, the file does not exist.

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build io.github.kitware.F3D

File: io.github.kitware.F3D.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.kitware.F3D</id>
```
Comment (nedrichards) [inline io.github.kitware.F3D.metainfo.xml:3]: is this in fact the best app-id? kitware.com exists and that's where they host the sourcecode. Does the app make use of an app-id internally currently?

Comment (kevinsmia1939) [reply io.github.kitware.F3D.metainfo.xml:3]: I don't find any file matching appdata or metadata, so I don't think there is.
They use gitlab to host source code, but it seems like they use github as there home page website, I will ask them to clarify on this.

Comment (barthalion) [reply io.github.kitware.F3D.metainfo.xml:3]: Yeah, `com.kitware.F3D` may be better here.

**Comment by kevinsmia1939** [comment]
bot, build com.kitware.F3D

**Comment by hfiguiere** [comment]
The manifest is gone.

**Comment by hfiguiere** [comment]
bot, build com.kitware.F3D

**Comment by kevinsmia1939** [comment]
bot, build com.kitware.F3D

**Pull request closed without merging**